jitpack.yml

```
@@ -0,0 +1,6 @@
+before_install:
+  - wget https://raw.githubusercontent.com/sormuras/bach/releases/11/install-jdk.sh
+  - source install-jdk.sh --feature 17
+  - jshell --version
```

It seems specifying `jdk` here should work as well:
```suggestion
jdk:
  - openjdk17
```
Maybe that is a bit more efficient in can JitPack has build images for these JDK versions prepared already?

Though it is not clear what distribution or what exact JDK 17 version this refers to. But should be fine nonetheless I think.

---

What confuses me a bit is that your JitPack config uses JDK 17, but the build script of this project specifies to use a JDK 11 toolchain.

And because I currently don't have toolchain auto-provisioning enabled, I think the build should have failed. So that suggests the build image also had JDK 11 installed (?).